server/bin/pbench-audit-server.py

```diff
@@ -89,52 +90,53 @@ def dump(self):
                 for fil in sorted(self.bad_controllers):
                     f.write("\t* {}\n".format(fil))
             for fil in sorted(self.controllers):
```

Comment:
Why is the variable called `fil`? We are looping over the set of controllers, so why not call it `controller`?

      for controller in sorted(self.controllers):

reads much better.

Reply:
ok